api/napi/src/interpreter/value.rs

```diff
@@ -105,12 +108,52 @@ pub fn to_value(env: &Env, unknown: JsUnknown, typ: Type) -> Result<Value> {
             Ok(Value::Bool(js_bool.get_value()?))
         }
         Type::Color => {
-            let js_color: JsExternal = unknown.coerce_to_object()?.get("color")?.unwrap();
-            Ok(Value::Brush(Brush::from(env.get_value_external::<Color>(&js_color)?.clone())))
+            let color_ref = env.create_reference(unknown.coerce_to_object()?)?;
+            if let Some(js_color) = env
+                .get_reference_value::<JsObject>(&color_ref)
+                .ok()
+                .and_then(|obj| obj.get("color").ok().flatten())
+                .and_then(|brush_prop| env.get_value_external::<Brush>(&brush_prop).ok())
+            {
+                return Ok(Value::Brush(js_color.clone()));
+            }
+
+            if let Some(js_brush) = env
+                .get_reference_value::<JsObject>(&color_ref)
+                .ok()
+                .and_then(|js_object| js_object.coerce_to_string().ok())
+                .and_then(|string| string_to_brush(string).ok())
+            {
+                return Ok(js_brush);
```

Comment:
Not strictly related to this patch, as this happened before as well, but when `to_value` is called with `Type::Color`, the function does not return a `Value::Color` but a `Value::Brush`. That seems wrong, and the nodejs bindings seems to have gotten it "right" by doing the opposite: Always returning a `Value::Color` even when a `Brush` is requested - although that conversion is safe to do.